unit_testing/myapp/models.py

```diff
@@ -6,5 +7,27 @@ class Person(models.Model):
     first_name = models.TextField(max_length=50)
     last_name = models.TextField(max_length=50)
 
+    cache_key = u'person_{}'
+
     def __unicode__(self):
         return "{name}".format(name=self.first_name+self.last_name)
```

Comment:
User return unicode strings `"u{first_name} {last_name}".format(first_name=self.first_name, last_name=self.last_name).strip()` The concatenation is not correct. It will remove the space between the first and last name.